t/03-auth.t

```diff
@@ -26,31 +26,47 @@ use OpenQA::Test::TimeLimit '10';
 use OpenQA::WebAPI::Auth::OAuth2;
 use Mojo::File qw(tempdir path);
 use Mojo::Transaction;
+use Mojo::URL;
+use MIME::Base64 qw(encode_base64url decode_base64url);
 
 my $t;
 my $tempdir = tempdir("/tmp/$FindBin::Script-XXXX")->make_path;
 $ENV{OPENQA_CONFIG} = $tempdir;
 
 sub test_auth_method_startup {
     my ($auth, @options) = @_;
-    my @conf = ("[auth]\n", "method = \t  $auth \t\n");
+    my @conf = ("[auth]\n", "method = \t  $auth \t\n", "[openid]\n", "httpsonly = 0\n");
     $tempdir->child("openqa.ini")->spurt(@conf, @options);
 
     $t = Test::Mojo->new('OpenQA::WebAPI');
     is $t->app->config->{auth}->{method}, $auth, "started successfully with auth $auth";
-    $t->get_ok('/login');
+    $t->get_ok('/login' => {"Referer" => "http://open.qa/tests/42"});
```

Comment:
But that page doesn't exist no matter if the network is available or not.
It is not used by the OpenID module (it is even hidden using base64).
Also in that case only the last test `redirect to original papge after login` would fail.

Reply:
I meant if the network is available one gets a 404 response and if the network is not available one gets a connection error. So this could have made a difference. But considering your other comment it doesn't so you can ignore my comment.